index.html

```diff
@@ -2595,6 +2595,133 @@ <h3>Data Schemas</h3>
+          <a>verifier</a> may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the JSON-LD
+          context(s) used in the <a>verifiable credential</a> are the
+          same when used by both the <a>issuer</a> and <a>verifier</a>. 
+        </p>
+        <p>
+          To validate that a resource referenced by a <a>verifiable
+          credential</a> is the same at verification time as it is at
+          issuing time, an implementer MAY include a property named
+          <code>relatedResource</code> that stores an array of objects
+          that describe additional integrity metadata about each
+          resource referenced by the <a>verifiable credential</a>. If
+          <code>relatedResource</code>
+          is present, there MUST be an object in the array for each remote
+          resource for each context used in the verifiable credential. 
+        </p>
+        <p class="issue" title="Mandatory listing of contexts in relatedResouce are under debate.">
+The requirement that contexts be listed in `relatedResource` is currently being debated in the VCWG. This requirement might be removed in future iterations of the specification. 
+        </p>
+        <p>
+          Each object in the
+          <code>relatedResource</code> array MUST contain the following:
+          the [[URL]] to the resource named <code>id</code> and the 
+          <code>digestSRI</code> information for the resource
+          constructed using the method specified in <a
+          href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource
+          Integrity</a>.
+        </p>
+        <p class="issue" title="Unification of cryptographic hash expression formats are under discussion">
+The Working Group is currently attempting to determine if cryptographic hash expression formats can be unified across all of the VCWG core specifications. Candidates for this mechanism include `digestSRI` and `digestMultibase`. There are arguments for and against unification that the WG is currently debating.
+        </p>
+          There MUST NOT be more than one object in the
+          <code>relatedResource</code> per <code>id</code>. 
+        </p>
+        <p>
+          An object in the <code>relatedResource</code> array MAY
+          contain a property named <code>mediaType</code> that indicates
+          the expected media type for the indicated
+          <code>resource</code>. If a <code>mediaType</code> is included
+          it SHOULD be a valid media type as listed in the 
+          <a href="https://www.iana.org/assignments/media-types/media-types.xhtml">
+            IANA Media Types
+          </a> registry.
+        </p>
+        <p>
+          Any object in the <a>verifiable credential</a>
+          that contains an `id` [[URL]] property MAY be annotated with 
+          integrity information as specified in this section.
```

Comment:
In the example `An example of a related resource integrity object in a  credentialSubject refering to an image`, there is no property named `relatedResource`. I found that confusing. Would it be helpful to clarify how such an object can be annotated?

Reply:
added in [4088c86](https://github.com/w3c/vc-data-model/pull/1140/commits/4088c86c40aade9a0d220c15a71e48e614a65ae6) - thanks!